white-paper/wp.tex

```diff
@@ -3,7 +3,258 @@
 \usepackage{float}
 \usepackage[bottom]{footmisc}
 \usepackage{bookmark}
-\usepackage{microtype}
+\usepackage{microtype}\documentclass[12pt,oneside]{article}
```

Comment:
@IlyaMoskva it seems to be a duplicate: we already have `documentclass` at the beginning

Reply:
@g4s8 It was copy-paste issue (I work in LaTeX online editor). I've dropped this commit and fixed again. It's strange, why build was successful.